user.rb

```diff
@@ -12,4 +12,10 @@ def get_hoge
   def poyo
     1 + 1
   end
+
+  def foo
+    hoge {
```

Comment:
__[development][rubocop]__
* Avoid using `{...}` for multi-line blocks. [:ref](https://github.com/bbatsov/ruby-style-guide#single-line-blocks)

<!-- Comment by [SideCI](https://sideci.com) -->